### ISSUE TITLE
iOS status bar tap fails to scroll scaffold

### ISSUE THREAD

**Issue by cbracken** [flutter/flutter#8428]
## Steps to Reproduce

On iOS:
1. In Flutter Gallery main page, scroll to bottom.
1. Tap status bar.

Expected: scrolls to top.
Actual: no scroll.

Relevant doctor info:
```
[✓] Flutter (on Mac OS, channel master)
    • Flutter at /Users/cbracken/src/flutter/flutter
    • Framework revision 97083ac6aa (46 minutes ago), 2017-02-27 12:24:02
    • Engine revision 0a7b177c33
    • Tools Dart version 1.23.0-dev.0.0

[✓] iOS toolchain - develop for iOS devices (Xcode 8.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 8.2, Build version 8C38
    • ios-deploy 1.9.1
```

**Comment by cbracken** [comment]
After a bit of digging this is specific to the example apps. The `primary` constructor parameter dictates which view is primary for scrolling.

Preparing a patch that adds the constructor arg where necessary.

**Comment by cbracken** [comment]
Fixed by #8506
